ramls/circulation.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Circulation
-version: v7.1
+version: v7.2
```

Comment:
This now needs updating to 7.3 as master has already moved on to 7.2 due to CIRC-196. Please merge master at the same time.

Reply:
Done